server/events/project_command_builder.go

```diff
@@ -28,6 +29,14 @@ const (
 	// DefaultWorkspace is the default Terraform workspace we run commands in.
 	// This is also Terraform's default workspace.
 	DefaultWorkspace = "default"
+	// DefaultAutomergeEnabled is the default for the automerge setting.
+	DefaultAutomergeEnabled = false
+	// DefaultAutoDiscoverEnabled is the default for the auto discover setting.
+	DefaultAutoDiscoverEnabled = true
+	// DefaultParallelApplyEnabled is the default for the parallel apply setting.
+	DefaultParallelApplyEnabled = false
+	// DefaultParallelPlanEnabled is the default for the parallel plan setting.
+	DefaultParallelPlanEnabled = false
```

Comment:
Should these other defaults be added in this PR? I can't see any reference to them in your new code.

Reply:
Nice catch! These got dragged in when I merged #3279 with this PR. They aren't relevant anymore so I've removed them.